integration-test/2022-disputed-borders.py

```diff
@@ -321,3 +321,104 @@ def test_places_with_viewpoints(self):
                 'kind:tr': None,  # invalid place type not converted to a kind
                 'kind:xx': None,  # invalid viewpoint not exported
             })
+
+    def test_boundary_administrative_has_dispute_id(self):
+        z, x, y = (16, 53533, 28559)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/relation/202058477
+            dsl.way(202058477, dsl.tile_diagonal(z, x, y), {
+                'admin_level': '2',
+                'boundary': 'administrative',
+                'dispute': 'yes',
+                'disputed_by': 'XX',
+                'name': 'Extent of Indian Claim at Bara Hotii Valleys',
+                'ne:brk_a3': 'B02',
+                'ne_id': '1746708469',
+                'type': 'linestring',
+                'source': 'openstreetmap.org',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'boundaries', {
+                'id': 202058477,
+                'kind': 'disputed_reference_line',
+                'dispute_id': 'B02_1746708469',
+                'kind:xx': 'unrecognized'
+            })
+
+    def test_ne_breakaway_has_dispute_id(self):
+        z, x, y = 8, 0, 0
+
+        self.generate_fixtures(
+            dsl.way(1, dsl.tile_diagonal(z, x, y), {
+                'featurecla': 'Breakaway',
+                'BRK_A3': '456',
```

Comment:
Nit: There may be a setting in the shapefile importing step to lowercase all field names? In DBF the field names are case insensitive. But then we might be tripping up ourselves later it looks like.

Reply:
fixed in [4624bff](https://github.com/tilezen/vector-datasource/pull/2079/commits/4624bff282298e346af4fb4f60dda5f1d1b86a7d)